Mathlib/Data/Finset/Lattice/Fold.lean

```diff
@@ -8,6 +8,7 @@ import Mathlib.Data.Multiset.Lattice
 import Mathlib.Data.Set.Lattice
 import Mathlib.Order.Hom.Lattice
 import Mathlib.Order.Nat
+import Mathlib.Data.Finset.Sum
```

Comment:
Pedantry: would you like to sort these imports alphabetically?

Reply:
Done!